common/cache/namespaceCache.go

```diff
@@ -30,16 +30,19 @@ import (
 	"sync/atomic"
 	"time"
 
-	namespacepb "go.temporal.io/temporal-proto/namespace"
+	"github.com/gogo/protobuf/proto"
+	"go.temporal.io/temporal-proto/namespace"
```

Comment:
Can you fix all proto imports to have alias and `pb` suffix. I did it this way everywhere (spent awhile to make it consistently through the entire project).